core/src/test/java/org/commonjava/maven/ext/core/impl/VersionTest.java

```diff
@@ -303,12 +303,12 @@ public void testIsSnapshot()
     @Test
     public void testIsEmpty()
     {
-        assertTrue( Version.isEmpty(null) );
-        assertTrue( Version.isEmpty("") );
-        assertTrue( Version.isEmpty("  \n") );
+        assertTrue( Version.isEmpty( null ) );
+        assertTrue( Version.isEmpty( "" ) );
+        assertTrue( Version.isEmpty( String.format( "  %n" ) ) );
```

Comment:
Why not just use `"  " + System.lineSeparator()` ?

Reply:
It could, it just looked closer to the original that way.

Reply:
I think I would prefer System.lineSeparator() if we are using that elsewhere (its then more consistent)

Reply:
Maybe it makes sense to use `%n` only if there's an existing `String.format()` call or equivalent there and `System.lineSeparatror()` otherwise? The one time I'd say `%n` is easier to use is when it appears in the middle of lines that would otherwise have to all be manually concatenated.

Reply:
That makes some sense. String concatenation in tests shouldn't matter (and we also have the option of using the SLF4J MessageFormatter class).